### PR TITLE
feat(security): Updates to the data encryption doc

### PR DESCRIPTION
**Pull request by paperclypse**
Imported from a forked repo created by @jaclowell.

### CODE REVIEW

File: src/content/docs/security/security-privacy/compliance/data-encryption.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ tags:
   - Security
   - Security and Privacy
   - Compliance
-metaDescription: "New Relic's data encryption methods, including who gets it, what data is encrypted, and how it works with data at rest or in transit."
+metaDescription: 'New Relic's data encryption methods, including who gets it, what data is encrypted, and how it works with data at rest or in transit.'
```
Comment (paperclypse) [inline src/content/docs/security/security-privacy/compliance/data-encryption.mdx:7]:
```suggestion
metaDescription: "New Relic's data encryption methods, including who gets it, what data is encrypted, and how it works with data at rest or in transit."
```

File: src/content/docs/security/security-privacy/compliance/data-encryption.mdx [hunk 2]
```diff
@@ -101,12 +101,7 @@ New Relic's disk-based encryption provides additional security while your data i
       </td>
 
       <td>
-        This encryption protects the physical disk where New Relic retains your data, including the following:
-
-        * [Log management](/docs/logs/new-relic-logs/get-started/new-relic-logs-security-privacy) data
-        * [Dimensional metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metric-time-series) data
-
-        As we implement encryption at rest for additional telemetry types, your data will be encrypted automatically with no additional steps required by you.
+        All data at rest
```
Comment (paperclypse) [inline src/content/docs/security/security-privacy/compliance/data-encryption.mdx:104]:
```suggestion
        Applicable data at rest
```

**Comment by paperclypse** [comment]
Looks like this is ready for a review @barbnewrelic

**Review by barbnewrelic — APPROVED** [review]
Thanks @jaclowell for making these important changes to this doc!

**Pull request merged**